src/appleseed/renderer/kernel/lighting/pathvertex.h

```diff
@@ -70,7 +70,7 @@ class PathVertex
     // Path properties.
     size_t                      m_path_length;
     int                         m_scattering_modes;
-    Spectrum                    m_throughput;
+    Spectrum                    m_throughput;   // Cumulative throughput (percent of incoming light which is reflected as outgoing light) of this path, up to but excluding this vertex
```

Comment:
Maybe indent the comment such that it's right of all variable names (i.e. `m_scattering_modes`).
Also the codeguidlines state
`Comments on the same line as code start with a lower case letter and don't end with a period.`